### ISSUE TITLE
Disable color contrast rule and add option to turn it on selectively

### ISSUE THREAD

**Issue by holdenhinkle** [department-of-veterans-affairs/va.gov-team#17257]
## Issue Description
Checking for color contrast accessibility issues every time `injectAxe()` is called takes time, is cpu-intensive, and isn't necessary to do on every page.

From conversation in Slack - https://dsva.slack.com/archives/C01CHAY3ULR/p1607007929214500

Holden Hinkle  10:05 AM
Trevor Pierce and I talked yesterday about the deque docs mentioning that checking for color-contrast can be quite cpu intensive and that they suggest the following if you’re checking it on lots of pages:

_**Targeted color-contrast analysis**
If you are analyzing multiple pages on a single Web site or application, chances are these pages all contain the same styles. It is therefore not adding any additional information to your analysis to analyze every page for color-contrast. Choose a small number of pages that represent the totality of you styles and analyze these with color-contrast and analyze all others without it._

I did a quick (haha not so quick--it takes ~25 mins to run all the Cypress tests on my machine) a/b comparison between running the tests checking `color-contraston` each page and not checking `color-contrast` on each page (by adding `'color-contrast': { enabled: false }` to the` rules` object in the options `object` that’s passed into` cy.checkA11y().

```
**First test (checking for color-contrast:**
  ✖  5 of 76 failed (7%)                      25:00      155      116        5       34        -  

**Second test (not checking for color-contrast:**
  ✖  7 of 76 failed (9%)                      21:56      155      113        8       34        -  
Shaved 3 mins 4 seconds off. (edited) 
```

---
## Tasks
- [ ] Disable `color-contrast` by default and make an option to selectively turn it on.

## Acceptance Criteria
- [ ] The `axeCheck()` Cypress custom command will be updated.

**Comment by trevor** [comment]
@ holdenhinkle Who's Pierce?

**Comment by holdenhinkle** [comment]
Sorry, that was a cut and paste issue. That's Trevor Pierce @1Copenut

**Comment by pjhill** [comment]
This ticket was addressed elsewhere and I believe what was originally outlined here has basically been [accomplished](https://github.com/department-of-veterans-affairs/vets-website/blob/104816b114c34ca21837b0dfec5df6be7bf73446/src/platform/testing/e2e/cypress/support/commands/axeCheck.js#L52). I'm closing this ticket.
